src/general/statistics_tracking/ReproductionStatistic.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Components;
+using DefaultEcs;
+using Newtonsoft.Json;
+
+public class ReproductionStatistic : IStatistic
+{
+    [JsonIgnore]
+    public StatsTrackerEvent LinkedEvent { get; set; } = StatsTrackerEvent.PlayerReproduced;
+
+    [JsonProperty]
+    public int TimesReproduced { get; set; }
+
+    [JsonProperty]
+    public Dictionary<Biome, int> ReproducedInBiomes { get; set; } = new();
+
+    [JsonProperty]
+    public Dictionary<string, int> ReproducedWithOrganelle { get; set; } = new();
+
+    public void RecordPlayerReporduction(in Entity player, Biome? biome)
```

Comment:
```suggestion
    public void RecordPlayerReproduction(in Entity player, Biome? biome)
```

Typo in the name. Also shouldn't the biome be always required here? Is there a situation where the player can reproduce without being in any biome?